### PR TITLE
Add Radix Sort

### PR DESCRIPTION
**Pull request by TawfikYasser**

### CODE REVIEW

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
code fails on the following case : - 

### Case : 
```
int arr[] = {1,2,3,4,5}
```

### your output 

```
1
2
3
4
<random garbage value>
```

**Comment by ming-tsai** [comment]
> code fails on the following case : - 
> 
> ### Case : 
> ```
> int arr[] = {1,2,3,4,5}
> ```
> 
> ### your output 
> 
> ```
> 1
> 2
> 3
> 4
> <random garbage value>
> ```

Hi @TawfikYasser,
Please make this recommendations

**Comment by TawfikYasser** [comment]
Hi @Arsenic-ATG , @ming-tsai Sorry for late reply, I have a lot of notifications, Please excuse me

For the code, I updated it, and sorry for the errors, I hope it now working good.

Thanks

File: sorting/c-or-cpp/radix-sort.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  for (int i = size - 1; i >= 0; i--) {
+    output[count[(array[i] / place) % 10] - 1] = array[i];
+    count[(array[i] / place) % 10]--;
+  }
+
+  for (int i = 0; i < size; i++)
+    array[i] = output[i];
+}
+
+void radixsort(int array[], int size) {
```
Comment (Arsenic-ATG) [inline sorting/c-or-cpp/radix-sort.cpp:35]: typo : -
you are calling `radixSort()` and defined `redixsort()` here ( 'S' capital in function call )

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Functionality wise it is working after fixing the typo

File: sorting/c-or-cpp/radix-sort.cpp (hunk 1 above)
Comment (TawfikYasser) [reply sorting/c-or-cpp/radix-sort.cpp:35]: Sorry for that. I will edit it now

**Comment by TawfikYasser** [comment]
Updated successfully

**Review by Arsenic-ATG — APPROVED** [review]

**Review by Arsenic-ATG — APPROVED** [review]

**Comment by Arsenic-ATG** [comment]
merge conflict resolved

**Comment by ming-tsai** [comment]
Hi @TawfikYasser,
Good job, thanks for your contributions, and welcome to our team
> The invitation [link](https://github.com/MakeContributions)

**Comment by TawfikYasser** [comment]
Thanks @ming-tsai , @Arsenic-ATG 

I’m glad to hear that, It’s a great community, I will contribute more to help other people. :smile:

**Comment by TawfikYasser** [comment]
I’m going to add most of data structures like: `Doubly linked list`, `Stacks`, `Queues`, and `Trees` soon, Because now I have exams.

**Pull request merged**